ApiSamples/Base.lproj/Localizable.strings

```diff
@@ -167,10 +167,10 @@
 "SampleDescription_PolygonWithElevation" = "Display a polygon on the map at a height above the ground";
 
 /* No comment provided by engineer. */
-"SampleTitle_AccuracyRingArroundBlueSphere" = "Blue sphere with accuracy ring arround";
+"SampleTitle_AccuracyRingAroundBlueSphere" = "Blue sphere with accuracy ring";
 
 /* No comment provided by engineer. */
-"SampleDescription_AccuracyRingArroundBlueSphere" = "Move the blue sphere with accuracy ring to a different point on the map";
+"SampleDescription_AccuracyRingAroundBlueSphere" = "Move the blue sphere with accuracy ring to a different point on the map";
```

Comment:
Description is not valid as the example has changed. Can we do something like
`Show an accuracy ring around blue sphere` instead